CONTRIBUTING.md

```diff
@@ -1,55 +1,77 @@
-# Contributing
+# How to Contribute to Spectrum Web Components
 
-Thanks for choosing to contribute!
+First off, thanks for taking the time to contribute!
 
-The following are a set of guidelines to follow when contributing to this project.
+The following is a set of guidelines for contributing to Spectrum Web Components which is hosted in the [Adobe Spectrum Web Components](https://github.com/adobe/spectrum-web-components) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
 
-## Code Of Conduct
+## I don't want to read this whole thing I just have a question!!!
 
-This project adheres to the Adobe [code of conduct](CODE_OF_CONDUCT.md). By participating,
-you are expected to uphold this code. Please report unacceptable behavior to
-[Grp-opensourceoffice@adobe.com](mailto:Grp-opensourceoffice@adobe.com).
+> **Note:** [Please don't file an issue to ask a question.](https://github.com/adobe/spectrum-web-components/discussions) You'll get faster results by using the resources below.
 
-## Have A Question?
+We have an official message board with a detailed FAQ and where the community chimes in with helpful advice if you have questions.
 
-Start by filing an issue. The existing committers on this project work to reach
-consensus around project direction and issue solutions within issue threads
-(when appropriate).
+-   [Github Discussions, the official Spectrum Web Components message board](https://github.com/adobe/spectrum-web-components/discussions)
+-   [Spectrum Web Components FAQ](https://github.com/adobe/spectrum-web-components/discussions/categories/q-a)
 
-## Contributor License Agreement
+## How Can I Contribute?
+
+### Contributor License Agreement
 
 All third-party contributions to this project must be accompanied by a signed contributor
-license agreement. This gives Adobe permission to redistribute your contributions
-as part of the project. [Sign our CLA](http://opensource.adobe.com/cla.html). You
-only need to submit an Adobe CLA one time, so if you have submitted one previously,
-you are good to go!
```

Comment:
This seems like important context about why you need to sign the CLA? Maybe we should keep some of this?

Reply:
I agree. The removed code of conduct and the emails for violations could probably be added here, as well.

Reply:
Done